### DISCUSSION TITLE
991CNX 中稳定的内存

### DISCUSSION THREAD

**Discussion by Physics365** [Physics365/991CN-X-CW-Decompilation#40] [Q&A]
991CNX 中稳定的内存有哪些（自检不清除）
是E9D4\~EDAB,F000\~F0FF吗

**Comment by Physics365** [comment]
据说F000后的内存不稳定，具体怎么体现

**Comment by Xyzstk** [comment]
屏幕缓冲区后的内存均不会清除,但若在重启后进行计算则栈至少会污染0xED70-0xEFFF的内存
0xF000-0xFFFF为SFR(特殊功能寄存器)，该区域可以通过与常规内存一致的指令进行读写，但用于操作硬件（如0xF800-0xFFFF用于控制屏幕），因此不能随意使用

**Reply by fish4terrisa-MSDSM** [reply]
这一段是不是可以插入到那个数据持久化的教程里？
